### ISSUE TITLE
App crashes with slide page transition

### ISSUE THREAD

**Issue by acidbiscuit** [nativescript-vue/nativescript-vue#406]
### Version
2.0.2

### Reproduction link
[https://github.com/acidbiscuit/nativescript-vue-slide-transition-crash](https://github.com/acidbiscuit/nativescript-vue-slide-transition-crash)
### Platform and OS info
Android 8.0 / NativeScript 5.1.0 / Ubuntu 18.04

### Steps to reproduce
Setup. Use 'slide' or 'slideLeft' transition for page's navigation  
Option 1
  - Open app
  - Press `Let's start` button to navigate to Dashboard
  - In `Accounts` tab press `Create` button
  - App crashes (logs in additional comments)
  
Option 2
  - Open app
  - Press `Let's start` button to navigate to Dashboard
  - In `Settings` tab press `Logout` button
  - App crashes (logs in additional comments)

### What is expected?
`slide` and `slideLeft` transitions should work consistently throughout the app, same as any other transition.

### What is actually happening?
Usage of `slide` and `slideLeft` page transition crashes the application 

---
Clearing and fresh installing the project does not resolve the issue. No additional conditions were found, except of direct usage of `slide` and `slideLeft` transitions. Though, it transition successfully works during navigation from initial route (Welcome.vue -> Dashboard.vue)

All other transitions were tested and do not cause any issues.

Don't have devices to reproduce the issue on iOS.

Device log after the crash:

```
android.content.res.Resources$NotFoundException: Unable to find resource ID #0xffffffec
at android.content.res.ResourcesImpl.getResourceTypeName(ResourcesImpl.java:265)
at android.content.res.Resources.getResourceTypeName(Resources.java:1989)
at android.support.v4.app.FragmentManagerImpl.loadAnimation(FragmentManager.java:1172)
at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1543)
at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625)
at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411)
at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366)
at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273)
at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:733)
at android.os.Handler.handleCallback(Handler.java:789)
at android.os.Handler.dispatchMessage(Handler.java:98)
at android.os.Looper.loop(Looper.java:171)
at android.app.ActivityThread.main(ActivityThread.java:6699)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:246)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:783)
```

`tns info` output:

```
✔ Getting NativeScript components versions information...
✔ Component nativescript has 5.1.0 version and is up to date.
✔ Component tns-core-modules has 5.1.1 version and is up to date.
✔ Component tns-android has 5.1.0 version and is up to date.
⚠ Update available for component tns-ios. Your current version is 5.0.0 and the latest available version is 5.1.0.
```

**Comment by MaxiSonntag** [comment]
Same issue with Nativescript-Angular and `fade` transition. Took me a whole lot of time to get behind this.
Maybe this is a general issue?

**Comment by acidbiscuit** [comment]
Guess it is general issue.

Related: https://github.com/NativeScript/NativeScript/issues/6810

**Comment by acidbiscuit** [comment]
Issue appears to be fixed in release [5.2.2](https://github.com/NativeScript/NativeScript/releases/tag/5.2.2)
